### PR TITLE
optimize: paramsPlaceHolder generate by StringUtils.repeat()

### PR DESCRIPTION
**Pull request by xianlaioy**
paramsPlaceHolder generate by StringUtils.repeat()

1.  paramsPlaceHolder generate by StringUtils.repeat()
2.  make read easy
3. run test LogStoreDataBaseDAOTest

### CODE REVIEW

**Comment by slievrly** [comment]
image
https://api.travis-ci.com/v3/job/345678226/log.txt

File: server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java [hunk 1]
```diff
@@ -165,15 +164,9 @@ public GlobalTransactionDO queryGlobalTransactionDO(long transactionId) {
...
             conn.setAutoCommit(true);
 
-            StringBuilder sb = new StringBuilder();
-            for (int i = 0; i < statuses.length; i++) {
-                sb.append("?");
-                if (i != (statuses.length - 1)) {
-                    sb.append(", ");
-                }
-            }
+            String paramsPlaceHolder = org.apache.commons.lang.StringUtils.repeat("?",",",statuses.length);
```
Comment (jsbxyyx) [inline server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:167]: extract `repeat` method to io.seata.common.util.StringUtils?

Comment (xianlaioy) [reply server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:167]: I don't think it's necessary.

Comment (jsbxyyx) [reply server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:167]: the `apache common langs` jar often conflicts with other libraries. convenient direct replacement.

File: server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java [hunk 2]
```diff
@@ -288,11 +281,9 @@ public boolean deleteGlobalTransactionDO(GlobalTransactionDO globalTransactionDO
     @Override
     public List<BranchTransactionDO> queryBranchTransactionDO(List<String> xids) {
         int length = xids.size();
-        int retsSize = length * 3;
-        List<BranchTransactionDO> rets = new ArrayList<>(retsSize);
-        StringJoiner sj = new StringJoiner(",");
-        xids.stream().forEach(xid -> sj.add("?"));
-        String sql = LogStoreSqlsFactory.getLogStoreSqls(dbType).getQueryBranchTransaction(brachTable, sj.toString());
+        List<BranchTransactionDO> rets = new ArrayList<>(length * 3);
+        String paramsPlaceHolder = org.apache.commons.lang.StringUtils.repeat("?",",",length);
```
Comment (slievrly) [inline server/src/main/java/io/seata/server/storage/db/store/LogStoreDataBaseDAO.java:285]: for better readability, please add a space in the separator.   `","`-> `", "`

**Comment by slievrly** [comment]
@xianlaioy In addition, you need to format and standardize the pr title.

**Pull request merged**